build.sh

```diff
@@ -178,7 +178,7 @@ function prepare_telegraf() {
     # generate certificate if doesn't exist
     gen_telegraf_cert
 
-    for a in ${swtiches[@]}; do
+    for a in ${switches[@]}; do
```

Comment:
thank you! I just realized lines 181-183 are actually for the debugging, these lines should be removed. do you mind removing them, I will merge the request.